orchestra/communication/utils.py

```diff
@@ -27,7 +27,8 @@ def mark_worker_as_winner(worker, task, required_role_counter,
     else:
         staffbot_request = staffbot_request.first()
 
-    staffbot_request.status = StaffBotRequest.Status.COMPLETE.value
+    staffbot_request.status = (
```

Comment:
shouldn't a winner mark the task as complete?

Reply:
you're right